src/natcap/invest/spec.py

```diff
@@ -457,25 +492,41 @@ def validate(self, filepath: str):
         Returns:
             A string error message if an error was found.  ``None`` otherwise.
         """
-        gdal_path = utils._GDALPath.from_uri(filepath)
-        if gdal_path.is_local:
-            file_warning = super().validate(filepath)
-            if file_warning:
-                return file_warning
+        with GDALUseExceptions():
```

Comment:
Adding the GDAL exceptions context manager to some functions that should probably have it, just in case.